### PR TITLE
Error boundary: error information

### PR DESCRIPTION
**Pull request by vasylshpak**
Add error boundary for Admin panel
- change Error Message to "Сталася помилка"

### CODE REVIEW

**Review by dieie32 — APPROVED** [review]

**Review by VitalikVoicix — APPROVED** [review]

**Review by nadiyafomenko — APPROVED** [review]

File: src/pages/error-page/error-page.js [hunk 1]
```diff
@@ -4,15 +4,14 @@ import Button from '@material-ui/core/Button';
...
+const { ERROR_BOUNDARY_STATUS } = config.statuses;
 
 const ErrorPage = () => {
   const classes = useStyles();
 
   return (
     <div className={classes.container}>
-      <h1>404</h1>
-      <h3>{ERROR_PAGE_STATUS}</h3>
+      <h2>{ERROR_BOUNDARY_STATUS}</h2>
```
Comment (moran711) [inline src/pages/error-page/error-page.js:14]: We have 2 errors `ERROR_PAGE_STATUS `and `ERROR_BOUNDARY_STATUS`. So why you remove `ERROR_PAGE_STATUS`?

Comment (vasylshpak) [reply src/pages/error-page/error-page.js:14]: As we discuss at Friday meeting it should display another text when an error comes up

Comment (vasylshpak) [reply src/pages/error-page/error-page.js:14]: @moran711 or you want to show different errors depends on which error type is coming up?

Comment (moran711) [reply src/pages/error-page/error-page.js:14]: It should display another error message on error page for error boundary, but not every time.

Comment (vasylshpak) [reply src/pages/error-page/error-page.js:14]: Got it, I'll make ternary operator than

Comment (vasylshpak) [reply src/pages/error-page/error-page.js]: done, thanks for comment

**Review by moran711 — APPROVED** [review]

**Pull request merged**